langchain/output_parsers/structured.py

```diff
@@ -0,0 +1,43 @@
+from __future__ import annotations
+
+import json
+from typing import List
+
+from pydantic import BaseModel
+
+from langchain.output_parsers.base import BaseOutputParser
+from langchain.output_parsers.format_instructions import STRUCTURED_FORMAT_INSTRUCTIONS
+
+line_template = '\t"{name}": {type}  // {description}'
+
+
+class ResponseSchema(BaseModel):
+    name: str
+    description: str
+
+
+def _get_sub_string(schema: ResponseSchema) -> str:
+    return line_template.format(
+        name=schema.name, description=schema.description, type="string"
```

Comment:
Why is type hardcoded?

Reply:
hardcoding to string for now, for simplicity